apps/etterlatte-gyldig-soeknad/src/main/kotlin/no/nav/etterlatte/JournalfoerInntektsjusteringService.kt

```diff
@@ -0,0 +1,152 @@
+        }
+    }
+
+    private fun opprettDokument(
+        tittel: String,
+        sakId: Long,
+        inntektsaar: String,
+        inntektsjustering: Inntektsjustering,
+    ): JournalpostDokument {
+        try {
+            logger.info("Oppretter PDF for inntektsjustering (id=${inntektsjustering.id})")
+
+            val pdf = opprettArkivPdf(sakId, inntektsaar, inntektsjustering)
+
+            return JournalpostDokument(
+                tittel = tittel,
+                dokumentvarianter = listOf(pdf),
+            )
+        } catch (e: ResponseException) {
+            throw Exception("Klarte ikke å generere PDF for inntektsjustering med id=${inntektsjustering.id}", e)
+        }
+    }
+
+    private fun opprettEksternReferanseId(
+        id: UUID,
+        sakType: SakType,
+    ): String = "etterlatte:${sakType.name.lowercase()}:inntektsjustering:$id"
+
+    private fun opprettArkivPdf(
+        sakId: Long,
+        inntektsaar: String,
+        inntektsjustering: Inntektsjustering,
+    ): DokumentVariant.ArkivPDF {
+        logger.info("Oppretter arkiv PDF for inntektsjustering med id ${inntektsjustering.id}")
+
+        return runBlocking {
+            retry {
+                pdfgenKlient.genererPdf(jsonInnhold(sakId, inntektsaar, inntektsjustering), "tom_mal")
+            }.let {
+                when (it) {
+                    is RetryResult.Success -> DokumentVariant.ArkivPDF(encoder.encodeToString(it.content))
+                    is RetryResult.Failure -> {
+                        logger.error("Kunne ikke opprette PDF for inntektsjustering med id ${inntektsjustering.id}")
+                        throw it.samlaExceptions()
+                    }
+                }
+            }
+        }
+    }
+}
+
+// TODO skal erstattes med egen mal
+private fun jsonInnhold(
+    sakId: Long,
+    aar: String,
+    inntektsjustering: Inntektsjustering,
+): JsonNode {
+    val json = """
+    {
+      "tittel" : "Inntektsjustering $aar",
+      "payload" : [
+        {
+          "type" : "heading-three",
+          "children" : [
+            {
+              "type" : "paragraph",
+              "text" : "Inntekt for sak $sakId ble mottatt ${inntektsjustering.tidspunkt}"
+            }
+          ]
+        },
+        {
+          "type" : "paragraph",
+          "children" : [
+            {
+              "type" : "paragraph",
+              "text" : "Arbeidsinntekt: ${inntektsjustering.arbeidsinntekt}"
+            },
+            {
+              "type" : "paragraph",
+              "text" : "Næringsinntekt: ${inntektsjustering.naeringsinntekt}"
+            },
+            {
+              "type" : "paragraph",
+              "text" : "Arbeidsinntekt utland: ${inntektsjustering.arbeidsinntektUtland}"
+            },
+            {
+              "type" : "paragraph",
+              "text" : "Næringsinntekt utland: ${inntektsjustering.naeringsinntektUtland}"
+            }
+          ]
+        }
+      ]
+    }
+"""
```

Comment:
Kanskje lage en egen mal for inntektsjustering? Da kan du bare sende `Inntektsjustering` til pdfgen og mappe der.

EDIT: Så at det er en todo på det i `AppBuilder.kt`